### ISSUE TITLE
Unity 2019.3.2 Error

### ISSUE THREAD

**Issue by DominicStewart** [Kamaell/beauty-and-the-beast#1]
Using this in newer versions doesn't seem to work. I think VFX now uses using UnityEngine.VFX; but not only that, there's a whole bunch of monobehaviour script errors.

**Comment by Kamaell** [comment]
Yes, I know there is a lot of changes now with now version. I'll check that tomorrow and try to update this to current version.

**Comment by DominicStewart** [comment]
Thank you! 

Fixing the namespace and removing the recorder seemed to fix the other script issues, but it would be worth checking the project and maybe updating for 2019.3.2 :)

By the way, im following along on your tutorial here:
https://medium.com/another-angle/beauty-and-the-beast-unity-vfx-graph-tutorial-aa8fe50dc3c2

I couldn't find your email so I thought I'd post this here:

I'm trying to make two skinned meshes completely made out of particles. One generates particles on a continuous basis, and the other sucks the generated particles in and that mesh's shape is formed once it has sucked in those particles. 

I've found a script that can generate attribute maps in real-time to be used for position and velocity for particles. This allows me to make a human entirely out of particles. Now I want to send those particles to my second human. I think I need to use the conform to sphere example you've made. This will allow the particles generated by the 1st model to stick and conform to the second models shape. 

However following along with your tutorial to the point where you test if the script is working with the sparking model gives me the following script error;

'Summoner' does not contain a definition for 'VFXInstance' and no accessible extension method 'VFXInstance' accepting a first argument of type 'Summoner' could be found (are you missing a using directive or an assembly reference?)	

Do you have any idea why this would happen?

**Comment by DominicStewart** [comment]
Nevermind, looking at the final script you didn't add

'private GameObject VFXInstance;' to the top.

**Comment by Kamaell** [comment]
I made update to version 2019.3.3. Unfortunately I had to set up this project from the start.
